packages/labs/context/src/lib/context-event.ts

```diff
@@ -0,0 +1,40 @@
+import {UnknownContext, ContextType} from './create-context.js';
+
+declare global {
+  interface HTMLElementEventMap {
+    /**
+     * A 'context-request' event can be emitted by any element which desires
+     * a context value to be injected by an external provider.
+     */
+    'context-request': ContextEvent<UnknownContext>;
+  }
+}
+
+/**
+ * A callback which is provided by a context requester and is called with the value satisfying the request.
+ * This callback can be called multiple times by context providers as the requested value is changed.
```

Comment:
Add `@param` jsdoc, especially for `dispose`

Reply:
Done, have not documented all functions yet, will do a doc pass after implemntation details settle.